mkdocs.yml

```diff
@@ -1,62 +1,25 @@
-# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
-
-site_name: ""
+site_name: "" # already part of the logo
```

Comment:
Ah, this is why all of our page titles (on browser tab) show up as "<Menu name> -"

If I enter Ask A Question here, we see "<Menu name> - Ask A Question"

Reply:
Can we change the logo to just the speech bubbles, and just use Ask A Question as site name? This will show on mobile version then. Right now, the top bar is empty in mobile version:

<img width="443" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/9a1af8f3-1a96-4026-a0e1-c03d42af25f9">

@Tanmay-97 thoughts?

Also in extra.css, the logo ratio is indeed off..  The logo is 1200 x 214 so some options are:
```css
.md-header__button.md-logo img {
  height: 43px;
  width: 240px;
  /* height: 36px; */
  /* width: 200px; */
}
```

Reply:
Yeah could do something like 
```
.md-header__button.md-logo img {
  height: 43px;
  aspect-ratio: 1200 / 214;
}
```

Reply:
Our 'official' logo is the one with the name on it; might get weird to just do speech bubbles. There should be a way to show only name in mobile version

Reply:
updated css file to correct the logo ratio, and have separate light vs dark mode strong font colors